content/dstu2/diagnostic-report.md

```diff
@@ -0,0 +1,26 @@
+---
+title: DiagnosticReport | FHIR DSTU 2 API
+---
+
+# DiagnosticReport
+
+* TOC
+{:toc}
+
+## Search
+
+Search for DiagnosticReports that meet supplied query parameters:
+
+    GET /DiagnosticReport?:parameters
+
+### Parameters
+
+Name | Type | Description
+-----|------|--------------
+`patient`|`string`| The subject of the report if a patient. Example: `12345`
```

Comment:
Update these types to be *reference* and link to FHIR documentation for reference types.  (See comment [here](https://github.com/cerner/fhir.cerner.com/pull/9#commitcomment-11149325).)